src/bin/server/state.rs

```diff
@@ -634,6 +643,7 @@ impl<'thr, 'store> ThreadState<'thr, 'store> {
             subscribed_db: None,
             sub_id: None,
             rx: None,
+            subscription_tx: subscription_tx,
```

Comment:
Just a style choice, but if the key is the same name as the variable you're assigning it to, you can just do `subscription_tx`.

Reply:
Thanks! Didn't know about this.